README.rst

```diff
@@ -1,6 +1,220 @@
+one application to handle multiple buckets at once.
+
+The passed `auth` parameter is a `requests <docs.python-requests.org>`_
+authentication policy, allowing you to authenticate with whatever means
+fits you.
+
+By default, Kinto supports Firefox Accounts and Basic authentication policies.
+
+.. code-block:: python
+
+    bucket = Bucket('payments', server_url='http://localhost:8888/v1',
+                    auth=credentials)
+
+    # Passing `create=True` to the bucket will make an HTTP request to
+    # create it.
+    bucket = Bucket('payments', server_url='http://localhost:8888/v1',
+                    auth=credential, create=True)
+
+
+Collections
+===========
+
+A collection is where records are stored.
+
+.. code-block:: python
+
+    # Once the bucket handy, use it to handle collections or groups.
+    collection = bucket.create_collection('receipts')
+
+    # Or get an existing one.
+    collection = bucket.get_collection('receipts')
+
+    # To delete an existing collection.
+    bucket.delete_collection('receipts')
+
+
+Records
+=======
+
+Records can be retrieved from and saved to buckets.
+
+.. code-block:: python
+
+    # Create an empty record
+    record = collection.create_record()
+
+    # You can also pass a python dictionary to represent the record
+    record = collection.create_record(dict(id='XXX', status='done',
+                                           title='Todo #1'))
+
+    # Get all records
+    record = collection.get_all_records()
+    record = collection.get_record(id='89881454-e4e9-4ef0-99a9-404d95900352')
+    collection.save_record(record)
+    collection.save_records([record1, record2])
+    collection.delete_record(id='89881454-e4e9-4ef0-99a9-404d95900352')
+    collection.delete_records([record1, record2])
+
+
+Handling conflicts
+------------------
+
+XXX
+
+Permissions
+===========
+
+ By default, the authenticated user will get read and write access to the
+ manipulated objects. It is possible to change this behavior by passing a dict
+ to the `permissions` parameter.
+
+ .. code-block:: python
+
+    record = collection.create_record(
+        data={},
+        permissions={'read': ['group:groupid']})
+
+.. note::
+
+    Every creation or modification operation on a distant object can be given
+    a `permissions` parameter.
+
+The `Bucket`, `Collection`, `Group` and `Record` class have a special
+`permissions` object that can be mutated in order to update the permissions
+model attached to the object.
+
+.. code-block:: python
+
+    bucket = Bucket('personal', auth=('alexis', 'p4ssw0rd'))
+
+    # XXX We need to find a way to get other's names from kinto, this isn't
+    # realistic.
+    friends = ['natim', 'niko', 'mat', 'tarek']
+    bucket.permissions.write += friends
+    bucket.permissions.create_collection += friends
+
+    # You *need* to call save in order to have these changes reflected in the
+    # remote.
+    bucket.permissions.save()
```

Comment:
permissions are attached to a root (bucket, group, record, collection), so it would update it there.